contrib/automation_tests/test_cases/capture_window.py

```diff
@@ -321,6 +334,15 @@ def _set_collect_thread_states_capture_option(self, collect_thread_states: bool,
             logging.info('Toggling "Collect thread states" checkbox')
             collect_thread_states_checkbox.click_input()
 
+    def _set_enable_auto_frame_track_capture_option(self, enable_auto_frame_track: bool,
+                                                    capture_options_dialog):
+        enable_auto_frame_track_checkbox = self.find_control('CheckBox',
+                                                             'AutoFrameTrackCheckBox',
+                                                             parent=capture_options_dialog)
+        if enable_auto_frame_track_checkbox.get_toggle_state() != enable_auto_frame_track:
+            logging.info('Toggling "Auto Frame Track" checkbox')
```

Comment:
Nit: It would be clearer if you specify the value to which the "Auto Frame Track" checkbox is being set in the log message instead of just logging "Toggling".

Reply:
This is consistent with all the other options. I will refactor them together in the next PR.